Index: Application/Class1.cs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>namespace Application;\r\n\r\npublic class Class1\r\n{\r\n    public static void Hello()\r\n    {\r\n        Console.WriteLine(\"hello world\");\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Application/Class1.cs b/Application/Class1.cs
--- a/Application/Class1.cs	(revision d40e2265c17bfdc5da43ae0df5e80dbfe0495a28)
+++ b/Application/Class1.cs	(date 1719434124639)
@@ -4,6 +4,6 @@
 {
     public static void Hello()
     {
-        Console.WriteLine("hello world");
+        Console.WriteLine("hello");
     }
 }
\ No newline at end of file
